### PR TITLE
fix wallet for new openssl

### PR DESCRIPTION
**Pull request by davidkelly**
The update to ssl caused wallet to crash, with a sad little `OPENSSL_Uplink(00007FF847238000,08): no OPENSSL_Applink`.  The fix is there in the openssl faq (https://www.openssl.org/docs/faq.html#PROG2).

### CODE REVIEW

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/3163/HighFidelity-Beta-PR11329-3e0b222ec45d10c21912ac9cf9d661924c8d97fd.exe

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/3171/HighFidelity-Beta-PR11329-c1422159350e7359cdddc79f791a5e462d549e0f.exe

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/3171/HighFidelity-Beta-PR11329-c1422159350e7359cdddc79f791a5e462d549e0f.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/3176/HighFidelity-Beta-PR11329-99cdbbe306aab40c67a992fd330e67d54fc63d4f.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/3176/HighFidelity-Beta-PR11329-99cdbbe306aab40c67a992fd330e67d54fc63d4f.exe

File: interface/resources/qml/hifi/commerce/purchases/Purchases.qml [hunk 1]
```diff
@@ -35,12 +35,20 @@ Rectangle {
     Hifi.QmlCommerce {
         id: commerce;
 
+        onAccountResult: {
+            if(result.status === "success") {
```
Comment (howard-stearns) [inline interface/resources/qml/hifi/commerce/purchases/Purchases.qml]: coding standard: space after if

**Comment by howard-stearns** [comment]
qa fail?  I set up a wallet and and tried to buy something, and got "Purcase Failed:..... invalid signature"

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/3184/HighFidelity-Beta-PR11329-0a3958f7433129b83aaae434af34b52597918f1d.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/3184/HighFidelity-Beta-PR11329-0a3958f7433129b83aaae434af34b52597918f1d.exe

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/3185/HighFidelity-Beta-PR11329-1903f65e45dcf7152ec9eeaa8b507a2dcc4910eb.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/3185/HighFidelity-Beta-PR11329-1903f65e45dcf7152ec9eeaa8b507a2dcc4910eb.exe

**Pull request merged**